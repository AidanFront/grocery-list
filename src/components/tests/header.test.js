import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from "../header";

afterEach(cleanup);

test('intergration rendering test of home page', () => {
  const { debug, container } = render(<Header />);

  expect(container.firstChild).toMatchSnapshot();
})