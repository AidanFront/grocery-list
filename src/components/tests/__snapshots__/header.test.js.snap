// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`intergration rendering test of home page 1`] = `
<header
  class="header"
>
  <div
    class="py-4 row"
  >
    <div
      class="col-12 col-md-6"
    >
      <i
        class="fas fa-carrot"
      />
      <h1
        class="pl-2 d-inline-flex"
      >
        Grocery List
      </h1>
    </div>
    <div
      class="pr-1 d-flex justify-content-md-center col-12 col-md-6"
    >
      <div
        class="c-search-bar position-relative"
      >
        <input
          class="rounded-pill px-2"
          id="search"
          placeholder="Search by Category"
          type="text"
          value=""
        />
        <i
          class="fas fa-search position-absolute"
        />
      </div>
    </div>
    <button
      class="c-add-item rounded-circle position-fixed"
      data-testid="add-button"
    >
      <i
        class="fas fa-plus"
      />
    </button>
  </div>
</header>
`;
